modules/ml/src/main/java/org/apache/ignite/ml/inference/storage/descriptor/ModelDescriptorStorage.java

```diff
@@ -45,4 +47,7 @@
      * @param mdlId Model identifier.
      */
     public void remove(String mdlId);
+
+    /** {@inheritDoc} */
+    public Iterator<IgniteBiTuple<String, ModelDescriptor>> iterator();
```

Comment:
Maybe we should clarify what entries iterator produces instead of just `@inheritDoc`?

Reply:
Good point, done.